src/main/java/Duke.java

```diff
@@ -1,10 +1,266 @@
+                    event.printEvent();
+                    saveTask(list);
+                }else if (input.startsWith("delete ")){
+                    Task task = makeDelete(list,input);
+                    printDelete(task,list.size());
+                    saveTask(list);
+                } else {
+                    throw new DukeException();
+                }
+            }catch (DukeTodoException e) {
+                printTodoError();
+            }catch (DukeDeadlineException e) {
+                printDeadlineError();
+            }catch (DukeEventException e) {
+                printEventError();
+            }catch (DukeException e) {
+                printError();
+            }catch (IOException e){
+                System.out.println("Writing to file failed.");
+            }
+        }
+        printBye();
+    }
+
+    private static void initialiseList() {
+        try {
+            getFile();
+        }catch(FileNotFoundException e){
+            try{
+                createFile();
+                System.out.println("File not found. Creating file...");
+            }catch(IOException e1){
+                System.out.println("Creating file failed.");
+            }
+        }
+    }
+
+    private static void getFile() throws FileNotFoundException {
+        Scanner scan = new Scanner(file);
+        while(scan.hasNextLine()){
+            String content = scan.nextLine();
+            String[] contents = content.split("\\s\\|\\s");
+            String legend = contents[0].trim();
+            Boolean done = Boolean.valueOf(contents[1].trim());
+            String action = contents[2].trim();
+            String action2 = "";
+            if(legend.equals("D") || legend.equals("E")){
+                action2 = contents[3].trim();
+            }
+            Task task;
+            if(legend.equals("T")){
+                list.add(new Todo(action,done));
+            }else if(legend.equals("D")){
+                list.add(new Deadline(action,done,action2));
+            }else if(legend.equals("E")){
+                list.add(new Event(action,done,action2));
+            }
+        }
+    }
+
+    private static void createFile() throws IOException{
+        Path path = Paths.get(FILENAME);
+        Files.createDirectory(path.getParent());
+    }
+
+    private static void printStart(){
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________\n" +
+                " Hello! I'm Duke\n" +
+                " What can I do for you?\n" +
+                "____________________________________________________________\n");
+    }
+
+    private static void printList(List<Task> list){
+        int num = 1;
```

Comment:
I think the name num here should indicating the index of item in the list plus one. Thus, it would be better if the name of the variable is index.